packages/circuit-ui/components/NotificationFullscreen/NotificationFullscreen.tsx

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright 2021, SumUp Ltd.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** @jsxRuntime classic */
+/** @jsx jsx */
+import { HTMLProps, MouseEvent } from 'react';
+import { jsx, css } from '@emotion/core';
+
+import Body from '../Body';
+import Headline from '../Headline';
+import ButtonGroup from '../ButtonGroup';
+import Button from '../Button';
+import { spacing } from '../../styles/style-mixins';
+import Image from '../Image';
+
+type Action = {
+  onClick?: (event: MouseEvent<HTMLButtonElement | HTMLAnchorElement>) => void;
+  href?: string;
+  text: string;
+};
+
+export interface NotificationFullscreenProps extends HTMLProps<HTMLDivElement> {
+  /**
+   * A concise description of the imageSrc prop.
+   */
+  imageSrc: string;
+  /**
+   * A concise description of the headline prop.
+   */
+  headline: string;
+  /**
+   * A concise description of the body prop.
+   */
+  body?: string;
+  /**
+   * A concise description of the actions prop.
+   */
+  actions: {
+    primary: Action;
+    secondary: Action;
+  };
+}
+
+const wrapperStyles = css`
+  max-width: 420px;
+  margin: 0 40px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
+const imageStyles = css`
+  height: 160px;
+  max-width: 280px;
+  object-fit: contain;
+`;
+
+const bodyStyles = css`
+  text-align: center;
+  margin-bottom: 0;
+`;
+
+/**
+ * Describe NotificationFullscreen here.
+ */
+export const NotificationFullscreen = ({
+  imageSrc,
+  headline,
+  body,
+  actions,
+  ...props
+}: NotificationFullscreenProps): JSX.Element => (
+  <div css={wrapperStyles} {...props}>
+    <Image css={imageStyles} src={imageSrc} alt="" />
+    <Headline css={spacing({ top: 'giga', bottom: 'byte' })} size="two">
+      {headline}
+    </Headline>
+    {body && <Body css={bodyStyles}>{body}</Body>}
+    <ButtonGroup css={spacing({ top: 'giga' })} align="center">
```

Comment:
Also: this doesn't quite match the designs, where on mobile the ButtonGroup's children should basically be inverted, with the primary action on top and the secondary one on the bottom:

<img width="703" alt="Screenshot 2021-06-04 at 18 25 35" src="https://user-images.githubusercontent.com/35560568/120833628-4a5ce180-c562-11eb-9c72-5b9ed1703ab5.png">

I started to implement it in here, but then realized that we'd also want this in the actual `ButtonGroup` element–so I think we can leave this to a future PR.

Reply:
Okay. Not sure if you saw my discussion about this with Giao-Chung in Figma. He insisted on having this, but I'm happy to add this in a follow-up PR.

Reply:
I haven't seen it–I think it's a really important thing to do, but it would probably make more sense to build it into the ButtonGroup and "inherit" the improvement in the NotificationFullscreen, I think